policy/modules/system/mount.te

```diff
@@ -182,6 +183,8 @@ optional_policy(`
 	corenet_dontaudit_udp_bind_all_reserved_ports(mount_t)
 	corenet_tcp_connect_all_ports(mount_t)
 
+	corecmd_exec_shell(mount_t)
```

Comment:
What requires this?

Reply:
I can't remember exactly what I did to produce the AVC for this, so unless I can reproduce it I'll drop it.

Reply:
This was needed by ZFS when running zedlets in `/usr/libexec/zfs`, but the new ZFS policy module makes this access obsolete.